src/themes.cpp

```diff
@@ -3074,8 +3078,9 @@ static void high_contrast_radio_round_down_box(int x, int y, int w, int h, Fl_Co
 
 static void high_contrast_hovered_up_frame(int x, int y, int w, int h, Fl_Color) {
 	fl_color(activated_color(fl_rgb_color(0xFF, 0xFF, 0xFF)));
-	fl_rect(x, y, w, h);
-	fl_rect(x+1, y+1, w-2, h-2);
+	fl_line_style(FL_SOLID, 2);
+	fl_rect(x+1, y+1, w-1, h-1);
+	fl_line_style(FL_SOLID);

```

Comment:
These 4 changes to drawing rects in the high contrast theme are necessary for the high contrast theme to look correct when high DPI screen scaling is in effect. Otherwise, there could be unexpected gaps between the two consecutive rectangles. This is fixed by drawing a single rectangle with a width of 2 instead.